optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc

```diff
@@ -0,0 +1,40 @@
+= OptaPlanner deprecates score DRL
+triceo
+2022-05-26
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: constraint, constraint streams, drl, deprecation
+
+Long story short: The time to switch to https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams] is now.
+Here's the https://www.optaplanner.org/learn/drl-to-constraint-streams-migration.html[score DRL migration guide].
+Score DRL is not going away in OptaPlanner 8.
+
+== The rise of Constraint Streams
+
+As I was starting with OptaPlanner more than a decade ago, score DRL was there, promising easy and fast score calculation.
+It wasn't as fast as the Incremental Java calculator, but it also wasn't as hard to write while still getting better results than the Easy Java calculator.
+On the other hand, it required me to learn a new programming language, and one that didn't (and still doesn't) have the best support in IDEs.
```

Comment:
Can we maybe remove this negative sentence? Or make this point in a positive way, by pointing out in the next paragraph that CS is just java (or kotlin), so has excellent IDE support (code highlighting, code completion, debugging)? 

I have an old blog with some screenshots that point that out: https://www.optaplanner.org/blog/2020/04/07/ConstraintStreams.html